src/main/java/org/aion/wallet/ui/components/HeaderPaneControls.java

```diff
@@ -12,31 +12,45 @@
 import javafx.scene.layout.VBox;
 import org.aion.log.AionLoggerFactory;
 import org.aion.log.LogEnum;
+import org.aion.wallet.connector.BlockchainConnector;
 import org.aion.wallet.dto.AccountDTO;
 import org.aion.wallet.ui.events.EventBusFactory;
 import org.aion.wallet.ui.events.EventPublisher;
 import org.aion.wallet.ui.events.HeaderPaneButtonEvent;
+import org.aion.wallet.ui.events.TimerEvent;
+import org.aion.wallet.util.BalanceFormatter;
+import org.aion.wallet.util.DataUpdater;
 import org.slf4j.Logger;
 
 import java.awt.*;
 import java.io.IOException;
+import java.math.BigInteger;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.net.URL;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Optional;
 import java.util.ResourceBundle;
 
 public class HeaderPaneControls implements Initializable {
 
     private static final Logger log = AionLoggerFactory.getLogger(LogEnum.WLT.name());
 
     private static final String AION_URL = "http://www.aion.network";
+
     private static final String STYLE_DEFAULT = "default";
+
     private static final String STYLE_PRESSED = "pressed";
+
     private static final double DEFAULT_ACCOUNT_WIDTH = 520;
+
     private static final double DEFAULT_BALANCE_WIDTH = 180;
 
+    private static final String CCY_SEPARATOR = " ";
```

Comment:
rename pls

Reply:
https://en.wiktionary.org/wiki/ccy
`CCY` is standard abreviation for currency :grimacing: